etl-data-pipeline/src/main/resources/application.properties

```diff
@@ -7,14 +7,18 @@ logging.level.gov.cdc.etldatapipeline=DEBUG
 # Apache Kafka Configurations
 spring.kafka.bootstrap-servers=localhost:9092
 spring.kafka.streams.application-id=debezium-cdc-consumer-app-1
+spring.kafka.producer.key-serializer=org.springframework.kafka.support.serializer.StringOrBytesSerializer
+spring.kafka.producer.value-serializer=org.springframework.kafka.support.serializer.JsonSerializer
 
 # Database
-spring.datasource.url=jdbc:sqlserver://cdc-nbs-alabama-rds-mssql.czya31goozkz.us-east-1.rds.amazonaws.com:1433;databaseName=rdb_test;encrypt=true;trustServerCertificate=true;
-spring.datasource.username=admin
-spring.datasource.password=${RDB_PASSWORD:}
+spring.datasource.url=${DB_URL:jdbc:sqlserver://localhost:1433;databaseName=NBS_ODSE;encrypt=true;trustServerCertificate=true;}
```

Comment:
Is this referencing a local db or this this meant as an alias?

Reply:
Yeah when we deploy we can set the prod db url in the parameter: `DB_URL`